templates/support/index.html

```diff
@@ -14,6 +14,7 @@ <h4>适用于企业级安全，支持和托管的开源支持服务</h4>
         <p>通过Canonical，可对从数据中心到容器再到数据库，LMA，服务器和云原生应用程序整个堆栈进行永不过时的验证。</p>
         <p>Ubuntu Advantage（企业订阅服务，简称“UA”）是一站式、每节点的支持方案，为最全面的企业开源基础设施和应用程序提供安全和支持服务，且托管服务已可用。</p>
         <p><a href="/support#ua-price" class="p-button--positive">价格列表</a><a href="/support#ua-srv" class="p-button--neutral">订阅服务内容</a></p>
+        <p>阅读 Ubuntu 企业订阅服务<a class="p-link--external" href="https://assets.ubuntu.com/v1/7c8938d9-UA-I-qa-list-v1.pdf">常见问题解答文档</a></p>
```

Comment:
Because it's in a dark background can you add `p-link--inverted` as a class too please?

Reply:
Thanks, updated.